helpers/kubernetes_cluster.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class Clover
+  def kubernetes_cluster_post(name)
```

Comment:
In general, helper methods should only be used when they reduce redundancy (when they are called from multiple places).  If you are only calling these helper methods from a single place, inline the logic at the callsite.

Reply:
We have only a single place where we call the post for now, but in the upcoming weeks we will add the update paths for this resource, similar to other resources like postgres or VM. Therefore, I think we can keep it as is. 

On the other hand, as far as I can see, offloading route logic to a `helper` became a pattern in code. IMO, obeying a pattern would be preferable, even if there's no benefit for such a case.

What do you think?

Reply:
If you are fairly sure you will be calling this from an additional place in the future, then it's OK to keep it in a helper.  However, be aware that similar helper methods in other helper files only exist because the methods are called from multiple places.  The general rule is that if you only need the logic in one place, put the logic in that place, do not add unnecessary indirection.

Reply:
Thank you, I will keep it then, as I'm sure we will have edits routes using this method.